packages/delegation/src/__tests__/index.test.ts

```diff
@@ -38,3 +47,316 @@ describe(accountIdToEthereumAddress, () => {
     ).toThrow('Invalid Chainflip account ID data');
   });
 });
+
+describe(DelegationSDK, () => {
+  let walletclient: WalletClient;
+  let publicClient: PublicClient;
+  let count = 0;
+  const calls: any[] = [];
```

Comment:
we push all the calls to the various wallets to this array so we can just snapshot one thing and get the sequence of calls